erizo_controller/erizoClient/src/utils/ConnectionHelpers.js

```diff
@@ -66,6 +74,14 @@ const GetUserMedia = (config, callback = () => {}, error = () => {}) => {
           screenConfig.video.mandatory = screenConfig.video.mandatory || {};
           screenConfig.video.mandatory.chromeMediaSource = 'desktop';
           screenConfig.video.mandatory.chromeMediaSourceId = config.desktopStreamId;
+          if (config.audio) {
+            screenConfig.audio = {
+              mandatory: {
+                chromeMediaSource: 'system',
```

Comment:
does it work if we use 'system' for audio and 'desktop' for video?

Reply:
Yes it worked for me.